new/LIT6280PrayerGeorge.xml

```diff
@@ -71,10 +71,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <div type="textpart" subtype="explicit" xml:lang="gez">
                         <note>This explicit is taken from <ref type="mss" corresp="MNC008"/>.</note>
                         <ab>ስብሐት፡ ለአብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ዘአንበሩ፡ ብየ፡ አክሊለ፡ ሞገሠ፡ መዘምረ፡ እኩን፡ ለጊዮርጊስ፡
-                            ኀያል፡ መስትጋድል፡ ኀያል፡ ኮከበ፡ ክብር፡ ኀያል፡ ገባሬ፡ ተአምር ፡ ጊዮርጊስ፡ ቅዱ<choice>
+                            ኀያል፡ መስተጋድል፡ ኀያል፡ ኮከበ፡ ክብር፡ ኀያል፡ ገባሬ፡ ተአምር ፡ ጊዮርጊስ፡ ቅዱ<choice>
                                 <sic>ሰ</sic>
                                 <corr resp="SH">ሱ</corr>
-                            </choice>፡ ለእግዚአብሔር፡ በእንተ፡ ጊዮርጊስ ፡ ሰማዕትከ ፡ ምህላነ ፡ ስማዕ፡ በዕዝንከ፡ ፫ ጊዜ፡ በል፡
+                            </choice>፡ ለእግዚአብሔር፡ በእንተ፡ ጊዮርጊስ ፡  ሰማዕትከ ፡  ምህላነ ፡ ስማዕ፡ በዕዝንከ፡ ፫ ጊዜ፡ በል፡
```

Comment:
Now I see one space before ፡ and two spaces after it (`ጊዮርጊስ-፡--ሰማዕትከ-፡--ምህላነ-፡-ስማዕ፡ `).
It should be `ጊዮርጊስ፡-ሰማዕትከ፡-ምህላነ፡-ስማዕ፡ `

Reply:
Thank you. I really have a problem to decipher space (one/two or even none) in Oxygen...

Reply:
one or no space matter, one or two spaces are really absolutely irrelevant :)